<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HCSDKWidget</class>
 <widget class="QWidget" name="HCSDKWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>海康SDK测试</string>
  </property>
  <widget class="QPushButton" name="btnConn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="log_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDeviveIP">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDevicePort">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>海康默认是8000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备连接IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备连接端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备连接用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDeviceUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备连接密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDevicePwd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoopTest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>轮询测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
